fn_phish_tank/fn_phish_tank/util/phish_tank_helper.py

```diff
@@ -0,0 +1,66 @@
+# (c) Copyright IBM Corp. 2010, 2019. All Rights Reserved.
+# -*- coding: utf-8 -*-
+
+import requests
+from requests.packages.urllib3.util import Retry
+from requests.adapters import HTTPAdapter
+import calendar
+from dateutil import parser
+
+TOTAL_RETRIES = 3
+BACK_OFF_FACTOR = 5
+
+
+class phish_tank_helper(object):
+    def __init__(self, req_session=None):
+        self.request_session = req_session
+
+    @staticmethod
+    def create_post_data(url, api_key=None):
```

Comment:
We should talk about using resilient-lib requests module and adding this logic for session retry. I'd like as much as possible to promote our own code rather than duplicating the logic per each integration.

Reply:
i have tried using resilient-lib requests module but, phinktank was not supporting for post call which library implements. actually by using library we pass the data in the `headers=headers, data=payload` format but phishtank api server is only support for `data=payload`. if we supplies `headers=headers` parameter it's giving an error. so not used resilient-lib request common module.

Reply:
we are making changes to support different ways to specify payload. Similar problems with the current scheme have come up.